english/eating-walls/light-fire/fire.md

```diff
@@ -0,0 +1,4 @@
+You build a camp-fire, grab a fistful of the wall and put it on a stick.
```

Comment:
@williamfbaur, it looks to me like you have two versions of the file - this one, and `make-some-smores.md`. To fix this, you can remove this file using the command `git rm`.